src/Common/src/CoreLib/System/HashCode.cs

```diff
@@ -54,6 +55,9 @@ namespace System
     public struct HashCode
     {
         private static readonly uint s_seed = GenerateGlobalSeed();
+#if BUILDING_COMPAT_HASHCODE // Will only be defined when building Microsoft.Bcl.HashCode project
+        private static bool UseNonRandomizedGlobalSeed => LocalAppContextSwitches.UseNonRandomizedHashSeed;
```

Comment:
Why double-cache the value? This is only ever read once, so it seems cleaner to just read the LocalAppContextSwitches value in the method and not define another field.

Reply:
Will fix